ui/src/lib.rs

```diff
@@ -23,24 +23,25 @@ enum Model {
 
 enum Msg {
     RouteChanged(Option<Route>),
-    StartGameMsg(page::start_game::Msg),
-    DemoMsg(page::demo::Msg),
-    LobbyMsg(page::lobby::Msg),
-    GameMsg(page::game::Msg),
+    Rendered(RenderInfo),
+    StartGame(page::start_game::Msg),
+    Demo(page::demo::Msg),
+    Lobby(page::lobby::Msg),
+    Game(page::game::Msg),
```

Comment:
Clippy did not like that these all ended in `Msg`. I guess constructors cant end in their own type name. Makes sense 👍